_docker/lib/rhd_environment.rb

```diff
@@ -72,7 +72,7 @@ def get_file(file_name)
   # that needs this is the Drupal pull-request environment
   #
   def template_resources
-    if environment_name == 'drupal-pull-request'
+    if is_drupal_environment?
```

Comment:
Can you explain the reason we need to template resources in any other environment other than the drupal-pull-request environment? As far as I can think of, the drupal-pull-request environment is the only one that would require some form of templating as we don't know the drupal or searchisko ports until run-time. 

Is there another scenario where we need to template?